package.json

```diff
@@ -108,7 +119,8 @@
     "mocha-only-detector": "1.0.0",
     "prettier": "^1.14.3",
     "rimraf": "^2.6.2",
-    "rsvp": "^4.8.4"
+    "rsvp": "^4.8.4",
+    "typescript": "^3.1.3"
```

Comment:
TypeScript does not follow SemVer, and [includes breaking changes](https://github.com/Microsoft/TypeScript/wiki/API-Breaking-Changes) in just about every middle-number release. You'll want this to be a `~` target as well, to avoid unknowningly taking in breaking changes.